### PR TITLE
Added variable stressor option for memory stress

### PR DESCRIPTION
**Pull request by harish-24**
Support memory stress with variable stressors and ability to run stressor list in parallel and in linear way

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

File: generic/stress-ng.py [hunk 1]
```diff
@@ -100,17 +103,18 @@ def test(self):
...
                             (self.class_type, self.workers))
         else:
-            for stressor in self.stressors.split(' '):
-                cmdline += '--%s %s ' % (stressor, self.workers)
-            args.append(cmdline)
+            if self.parallel:
+                for stressor in self.stressors.split(' '):
+                    cmdline += '--%s %s ' % (stressor, self.workers)
+                for v_stressor in self.v_stressors.split(' '):
+                    cmdline += '--%s %s ' % (stressor, self.workers)
```
Comment (narasimhan-v) [inline generic/stress-ng.py]: v_stressor ?

Comment (harish-24) [reply generic/stress-ng.py]: yeah, got that wrong.

**Comment by harish-24** [comment]
@narasimhan-v Pushed the changes. Thanks.

**Comment by PraveenPenguin** [comment]
@harish-24 thanks for patch , merging as already one ACK for this PR , @narasimhan-v thanks for ACK

**Pull request merged**